### DISCUSSION TITLE
Why are the results improved greatly since 2024-03-28?

### DISCUSSION THREAD

**Discussion by ShreckYe** [TechEmpower/FrameworkBenchmarks#9367] [Q&A] [answered]
The results with the run IDs [59c85cf5-37ff-48d5-8407-4f09bab0d538 (on 2024-03-28)](https://www.techempower.com/benchmarks/#section=test&runid=59c85cf5-37ff-48d5-8407-4f09bab0d538&hw=ph&test=fortune) and [cdec9eaf-19ea-48d2-bfa4-df15afbe3236 (on 2024-04-04)](https://www.techempower.com/benchmarks/#section=test&runid=cdec9eaf-19ea-48d2-bfa4-df15afbe3236&hw=ph&test=fortune) show significant increase compared to the previous result with run ID [da8306a2-d1d8-4bd3-95d4-59d8cd7047af (on 2023-12-02)](https://www.techempower.com/benchmarks/#section=test&runid=da8306a2-d1d8-4bd3-95d4-59d8cd7047af&hw=ph&test=fortune), with some results doubled or even tripled. The improvement has stayed since then. And there was also a 4-month gap. What's the reason behind this? Was there a hardware upgrade?

**Comment by volyrique** [comment, marked as answer]
Yes, there was; details in issue #8736.

**Comment by joanhey** [comment]
Perhaps will be good to add some information about server change in some place of the TFB results.
